clang/include/clang/CIR/CIRGenerator.h

```diff
@@ -55,6 +55,7 @@ class CIRGenerator : public clang::ASTConsumer {
   void Initialize(clang::ASTContext &astContext) override;
   bool HandleTopLevelDecl(clang::DeclGroupRef group) override;
   mlir::ModuleOp getModule() const;
+  mlir::MLIRContext &getContext() { return *mlirContext; }
```

Comment:
```suggestion
  mlir::MLIRContext &getMLIRContext() { return *mlirContext; }
```

MINOR preference?  We have all sorts of `contexts` all over the place that these are incredibly confusing.

Reply:
That's reasonable. I frequently find myself wondering which context things are returning (recently it's been the AST context coming up during IR generation).

Reply:
A little while back I changed all the variable, parameter, and field names in the ClangIR code to be `astContext` or `mlirContext`, getting rid of all generic `context` or `cxt` names.  I didn't do the same for function names.  But it would be good if those were also changed.